### ISSUE TITLE
NTR: Ethylene glycol present in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8263]
New term request Ethylene glycol present in blood (elthat_100703163324):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000568
- label: Elevated blood ethylene glycol concentration
- parents: [Blood xenobiotic (HP:0410172)](https://hpo.jax.org/app/browse/term/HP:0410172)
- pmid: PMID:30725694
- def: The concentration of ethylene glycol in the blood is above the upper limit of a guidance value for total concentrations that implies possible injurious effects.
- comment: Ethylene glycol (C2H6O2) is a toxic alcohol that is found in various household and industrial agents including antifreeze. Exposures are generally observed due to accidental or intentional ingestions, with its sweet taste lending to accidental toxic exposures, whereas intentional exposures may be motivated by a suicide attempt or desire for inebriation in the absence of ethanol.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8263
- synonyms:
